Sources/Kommunicate/Classes/Kommunicate.swift

```diff
@@ -440,6 +444,50 @@ open class Kommunicate: NSObject, Localizable {
         })
     }
     
+    /**
+     Creates and launches the conversation based on zendesk session.
+
+     - Parameters:
+     - viewController: ViewController from which the group chat will be launched.
+     */
+    open class func openZendeskChat(from: UIViewController) {
```

Comment:
Is it Zendesk or Zopim? Also, update the description.

Reply:
zopim is old version's name.we are using the latest sdk..They renamed it to `Zendesk Chat`